mentor_classifier_api/tests/test_followups.py

```diff
@@ -37,6 +37,12 @@ def test_followup(client, category, expected_results):
     with open(fixture_path("graphql/{}.json".format("category_answers"))) as f:
         data = json.load(f)
         responses.add(responses.POST, "http://graphql/graphql", json=data, status=200)
+    with open(fixture_path("graphql/{}.json".format("mentor_answers"))) as f:
```

Comment:
I don't think this needs the formatting? I would just do

```python
with open(fixture_path(path.join("graphql", "mentor_answers.json))) as f:
```

...we're not really supporting windows, but might as well stay platform agnostic w file paths